types/v56/vm_types.go

```diff
@@ -29,6 +29,8 @@ type Vm struct {
 	Deployed                bool   `xml:"deployed,attr,omitempty"`                // True if the virtual machine is deployed.
 	NeedsCustomization      bool   `xml:"needsCustomization,attr,omitempty"`      // True if this virtual machine needs customization.
 	NestedHypervisorEnabled bool   `xml:"nestedHypervisorEnabled,attr,omitempty"` // True if hardware-assisted CPU virtualization capabilities in the host should be exposed to the guest operating system.
+	// TODO delete
```

Comment:
Should this be cleaner up?

Reply:
Yep, cleared up, it's an error on my side, was a debug message